CONTRIBUTING.md

```diff
@@ -0,0 +1,184 @@
+# Contribution Guidelines
+
+Thank you for expressing interest in contributing to the development and
+maintenance of Crust firmware. The success of the Crust firmware project is
+made possible by community support of source code and documentation. This
+document contains guidelines and procedures for contributing to the Crust
+firmware project.
+
+## Code of Conduct
+
+This project is governed by the [Contributor Covenant Code of Conduct][coc].
+By contributing to Crust firmware, you are entering an agreement to uphold all
+guidelines and responsibilities outlined within. Please carefully read and
+review the agreement before contributing.
+
+## Issue Tracking
+
+All bugs and features are tracked through the Crust firmware GitHub [issues
+page][issues]. When filing issues, please follow the provided issue template to
+ensure prompt and personalized support.
+
+### Labels
+
+Labels are used to indicate the lifecycle of all Crust firmware issues. Labels
+are prefixed with letters corresponding to the chart below.
+
+| Label/Prefix     | Description
+|------------------|------------------------------------------------------------
+| Area (A)         | Indicates the area of the area of the project impacted.
+| Difficulty (D)   | Indicates the difficulty of an issue.
+| good first issue | Indicates issues that are suitable for new contributors.
+| help wanted      | Maintainers request community help resolving this issue.
+| Resolution (R)   | Indicates the resolution of an issue.
+| Status (S)       | Indicates the status of an issue.
+| Type (T)         | Indicates the type of an issue.
+
+## Submitting a Patch
+
+The Crust firmware project accepts patches through the use of GitHub pull
+requests. Although project maintainers accept impromptu pull requests on
+occasion, it is recommended that all pull requests correspond to an issue on
+the Crust firmware [issues page][issues]. Pull requests submitted _without_ a
+corresponding issue may be immediately rejected. If you believe Crust firmware
+could benefit from a new feature, please submit a feature request on the Crust
+firmware [issues page][issues].
+
+### Patch Lifecycle
+
+1. Comment on an issue from the Crust firmware [issues page][issues] to let
+   project maintainers know you would like to work on the issue. Project
+   maintainers will either acknowledge your comment or suggest another issue
+   if the issue is already being addressed.
+2. Submit a work-in-progress pull request (WIP) to receive early feedback from
+   project maintainers. Work-in-progress pull requests should be prefixed with
+   `WIP:` so project maintainers can assign labels to indicate the status.
```

Comment:
Are we using a label or a prefix? I guess non-members don't have access to labels

Reply:
> I guess non-members don't have access to labels

I assume this is the case. We can still assign labels after the prefix is added (unless we want to do away with the labels, WIP: is still a standard practice).

Reply:
ok, prefix it is then